app/views/customer_service/index.html.erb

```diff
@@ -208,77 +209,80 @@
+            <div class="usa-grid-full">
+              <div class="usa-width-one-fourth">
+                <h5><%= branch.name %></h5>
+              </div>
+              <div class="usa-width-three-fourths">
+                <table>
+                  <tbody>
+                    <% if contact.claims_dsn %>
+                      <tr>
+                        <th scope="row">DSN:</th>
+                        <td><%= contact.claims_dsn %></td>
+                      </tr>
+                    <% end %>
+                    <% if contact.claims_tel_comm %>
+                      <tr>
+                        <th scope="row">Comm:</th>
+                        <td><%= link_to contact.claims_tel_comm, "tel:#{contact.claims_tel_comm}" %></td>
+                      </tr>
+                    <% end %>
+                    <% if contact.claims_tel_tollfree %>
+                      <tr>
+                        <th scope="row">Toll-Free:</th>
+                        <td><%= link_to contact.claims_tel_tollfree, "tel:#{contact.claims_tel_tollfree}" %></td>
+                      </tr>
+                    <% end %>
+                    <% if contact.claims_email %>
+                      <tr>
+                        <th scope="row">Email:</th>
+                        <td><%= link_to contact.claims_email.sub(',', '<br>').html_safe, "mailto:#{contact.claims_email}" %></td>
+                      </tr>
+                    <% end %>
+                    <% if contact.claims_url %>
+                      <tr>
+                        <th scope="row">Online:</th>
+                        <td><%= link_to contact.claims_url, contact.claims_url, class: 'usa-external_link' %></td>
+                      </tr>
+                    <% end %>
+                    <% if contact.claims_fax_dsn %>
+                      <tr>
+                        <th scope="row">DSN Fax:</th>
+                        <td><%= contact.claims_fax_dsn %></td>
+                      </tr>
+                    <% end %>
+                    <% if contact.claims_fax_comm %>
+                      <tr>
+                        <th scope="row">Comm Fax:</th>
+                        <td><%= contact.claims_fax_comm %></td>
+                      </tr>
+                    <% end %>
+                    <% if contact.claims_fax_tollfree %>
+                      <tr>
+                        <th scope="row">Toll-Free Fax:</th>
+                        <td><%= contact.claims_fax_tollfree %></td>
+                      </tr>
+                    <% end %>
+                    <% if contact.claims_post %>
+                      <tr>
+                        <th scope="row">Address:</th>
+                        <td><%= simple_format(contact.claims_post) %></td>
+                      </tr>
+                    <% else %>
+                      <tr>
+                        <td></td>
+                        <td><p></p></td>
+                      </tr>
+                    <% end %>
```

Comment:
What's going on with this `else` block rendering an empty table row? Is that necessary?

Reply:
I didn't even notice that. I only added the outer loop and changed the indentation here. Looking at the customer service page UI it probably adds that space between each service without creating separate tables? None of them have a claims address right now but if they did it seems like that space wouldn't be there then... @jamesatheyDDS may know, he wrote the code

Reply:
Might be better to create separate tables, but I don't want to jam that into this PR

Reply:
It was a while ago. As I recall, it was to maintain a stable design when the information was missing. If it matters, I can dig into this again and tell you for certain.